app/src/main/kotlin/com/intive/picover/common/text/PicoverOutlinedTextField.kt

```diff
@@ -0,0 +1,84 @@
+package com.intive.picover.common.text
+
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import com.intive.picover.common.validator.ValidationStatus
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun PicoverOutlinedTextField(
+	modifier: Modifier = Modifier,
+	value: String,
+	label: String,
+	onValueChange: (String) -> Unit,
+	maxLines: Int = 1,
+	keyboardType: KeyboardType = KeyboardType.Text,
+	isError: Boolean = false,
+	errorText: String? = null,
+	isEnable: Boolean = true,
+	trailingIcon: @Composable (() -> Unit)? = null,
+) {
+	val keyboardController = LocalSoftwareKeyboardController.current
+	OutlinedTextField(
+		modifier = modifier,
+		value = value,
+		label = {
+			Text(text = label)
+		},
+		onValueChange = onValueChange,
+		maxLines = maxLines,
+		keyboardOptions = KeyboardOptions.Default.copy(
+			imeAction = ImeAction.Done,
```

Comment:
would be nice to have possibility to change imeAction :) for example you have several fields, so the last field would have `ImeAction.Done`, but for the rest fields probably `ImeAction.Next` would be better option

Reply:
@bartek977 any further comments?